### ISSUE TITLE
Switching from themes does not update the style correctly

### ISSUE THREAD

**Issue by mariusalexander** [dlr-gtlab/gtlab-core#1174]
### Summary

On windows I have noticed that when switching from dark to bright mode the style is still the "Fusion" style instead of the expected "windowsvista" style. A restart fixes this

### Expected Behaviour

The styles should update correctly.

### Reproduction steps

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

### Screenshots

From dark to bright (on windows!) 

![grafik](https://github.com/dlr-gtlab/gtlab-core/assets/52258575/0c53818d-1df1-4903-9308-b4a63edfec67)

### GTlab environment

```bash
GTlab Version: 2.0.6-dev
```

### OS

Windows

**Comment by jensschmeink** [comment]
@mariusalexander @rainman110 Should it be fixed in 2-0-7 or later?

**Comment by rainman110** [comment]
I think it is possible to fix in 2.0.7. The difficulty lies in creating a non-leaking but still working implementation. ;)

**Comment by rainman110** [comment]
I implemented a fix. See PR #1177
